python/pyspark/sql/connect/session.py

```diff
@@ -643,6 +656,13 @@ def addArtifacts(
 
     addArtifact = addArtifacts
 
+    def _cacheLocalRelation(self, localRelation: LocalRelation) -> str:
```

Comment:
I would name it as below: 
```suggestion
    def _cache_local_relation(self, localRelation: LocalRelation) -> str:
```